Firestore/Source/Public/FIRGetOptions.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Please add this header to the FirebaseFirestore.h umbrella header.

Reply:
Done.